guide/src/class/protocols.md

```diff
@@ -58,11 +58,28 @@ given signatures should be interpreted as follows:
     ```
     </details>
 
+  - `__lt__(<self>, object) -> object`
+  - `__le__(<self>, object) -> object`
+  - `__eq__(<self>, object) -> object`
+  - `__ne__(<self>, object) -> object`
+  - `__gt__(<self>, object) -> object`
+  - `__ge__(<self>, object) -> object`
+
+    The implementations of Python's rich-comparison operators `<`, `<=`, `==`, `!=`, `>` and `>=` respectively.
```

Comment:
Aren't these just "comparison operators"? To what property does "rich" refer to here?

Reply:
This is quoting directly from what Python calls them at https://docs.python.org/3/reference/datamodel.html#object.__ge__

(Well, they write it "rich comparison" so I'll adjust the wording to be the same.)